style.css

```diff
@@ -91,16 +109,449 @@
     background-color: #1a2236;
     color: white;
   }
-}
 
-.combo {
-  position: absolute;
-  width: 100%;
-  height: 160px;
-  left: 0;
-  top: 472px;
-  display: flex;
-  background-size: cover;
-  background-repeat: no-repeat;
-  background-image: url(/Asset/colorcombo.svg);
+  .combo {
+    position: absolute;
+    width: 100%;
+    height: 160px;
+    left: 0;
+    top: 472px;
+    display: flex;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-image: url(/Asset/colorcombo.svg);
+  }
+
+  .work-station {
+    display: grid;
+  }
+
+  .grid-container {
+    display: grid;
+    gap: 10px;
+    width: 100%;
+    height: 342px;
+    left: 0;
+    top: 30px;
+  }
+
+  .grid-item {
+    position: relative;
+    display: grid;
+    width: 100%;
+    height: 156px;
+    margin-top: 200px;
+    top: 285px;
+    background: #1a2236;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+    backdrop-filter: blur(0.5px);
+  }
+
+  .mac {
+    position: absolute;
+    display: flex;
+    width: 100%;
+    height: 186px;
+    left: 0;
+    top: 742px;
+    background: url(/Asset/mac1.svg);
+    background-repeat: no-repeat;
+    background-size: 100% auto;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  }
+
+  .mac2 {
+    position: absolute;
+    display: flex;
+    width: 100%;
+    height: 200px;
+    left: 0;
+    top: 1090px;
+    background: url(/Asset/mac2.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  }
+
+  .mac3 {
+    position: absolute;
+    display: flex;
+    width: 100%;
+    height: 186px;
+    left: 0;
+    top: 1438px;
+    background: url(/Asset/mac3.svg);
```

Comment:
- **[OPTIONAL]** I suggest using the images as requested by @adel-gu this will help you to fix the overlapping of the about section in the works section 👇🏽 

> [OPTIONAL] for better code structure and since the project item Image is part of the content, II highly recommend including each project img inside its own project grid item 🤔.